src/Apps/Artwork/Components/Seo/SeoDataForArtwork.tsx

```diff
@@ -124,53 +122,39 @@ export const SeoDataForArtworkFragmentContainer = createFragmentContainer(
   }
 )
 
-const displayPrice = (artwork: SeoDataForArtwork_artwork) => {
-  const {
-    is_price_hidden,
-    is_price_range,
-    listPrice,
-    sale_message,
-    price_currency,
-  } = artwork
-
-  if (is_price_range && !is_price_hidden && listPrice) {
-    return buildPriceSpecification(
-      price_currency,
-      splitPriceRange(listPrice.display)
-    )
-  }
-
-  if (sale_message && sale_message.includes("-")) {
-    return buildPriceSpecification(
-      price_currency,
-      splitPriceRange(sale_message)
-    )
-  }
-
-  return {
-    price: sale_message,
-    priceCurrency: price_currency,
-  }
-}
-
-const splitPriceRange = (priceRange: string) => {
-  const minAndMaxPrice = priceRange.split("-")
-  return {
-    minPrice: trim(minAndMaxPrice[0]).replace("$", ""),
-    maxPrice: trim(minAndMaxPrice[1]),
+export const offerAttributes = (artwork: SeoDataForArtwork_artwork) => {
```

Comment:
This function I copied from artist page without changing. I created a ticket to revisit to either put this along with query fragment in `<Product>` component or just reuse this function from a util file.